src/hooks/useWatchPendingTxs.ts

```diff
@@ -28,20 +29,26 @@ export const useWatchPendingTransactions = ({ address }: { address: string }) =>
   const { nativeCurrency, accountAddress } = useAccountSettings();
 
   const refreshAssets = useCallback(
-    (tx: RainbowTransaction) => {
-      if (tx.type === 'swap') {
-        // update swap assets
-        //swapRefreshAssets(tx.nonce);
-      } else {
-        // fetch assets again
-        fetchUserAssets({
-          address: accountAddress,
+    (_: RainbowTransaction) => {
+      // NOTE: We have two user assets stores right now, so let's invalidate both queries and trigger a refetch
+      const connectedToHardhat = getIsHardhatConnected();
+      queryClient.invalidateQueries(
```

Comment:
same here, split these out and added nft invalidation in case the user minted something.